invitations/tests/allauth/test_allauth.py

```diff
@@ -20,6 +21,8 @@ def setUpClass(cls):
             password='password')
         cls.invitation = Invitation.create(
             'email@example.com', inviter=cls.user)
+        cls.invitation.sent = timezone.now()
+        cls.invitation.save()
```

Comment:
The allauth tests were failing for me without adding a `sent` date, probably because we're not explicitly checking `key_expired()` instead of using the manager to do it.